src/applications/caregivers/components/PreSubmitInfo/index.jsx

```diff
@@ -170,62 +124,62 @@ const PreSubmitCheckboxes = ({
+          Primary Family Caregiver statement of truth
+        </h3>
 
-          <p className="vads-u-margin-y--4">
-            I certify that I am at least 18 years of age.
-          </p>
+        <p className="vads-u-margin-y--4">
+          I certify that I am at least 18 years of age.
+        </p>
 
-          <p>
-            I certify that I am a family member of the Veteran or service member
-            named in this application or I reside with the Veteran or service
-            member or will do so upon approval.
-          </p>
+        <p>
+          I certify that I am a family member of the Veteran or service member
+          named in this application or I reside with the Veteran or service
+          member or will do so upon approval.
+        </p>
+
+        <p>
+          I agree to perform personal care services as the Primary Family
+          Caregiver for the Veteran or service member named on this application.
+        </p>
 
-          <p>
-            I agree to perform personal care services as the Primary Family
-            Caregiver for the Veteran or service member named on this
-            application.
-          </p>
+        <p>
+          I understand that the Veteran or Veteran’s surrogate may initiate my
+          revocation as a Primary Family Caregiver at any time and that the VA
+          may immediately revoke this designation if I fail to comply with the
+          program requirements for continued participation in the program.
+        </p>
 
-          <p>
-            I understand that the Veteran or Veteran’s surrogate may initiate my
-            revocation as a Primary Family Caregiver at any time and that the VA
-            may immediately revoke this designation if I fail to comply with the
-            program requirements for continued participation in the program.
-          </p>
+        <p>
+          I understand that participation in the Program of Comprehensive
+          Assistance for Family Caregivers does not create an employment
+          relationship with the Department of Veterans Affairs.
+        </p>
 
-          <p>
-            I understand that participation in the Program of Comprehensive
-            Assistance for Family Caregivers does not create an employment
-            relationship with the Department of Veterans Affairs. I have read
-            and accept the
-            <a
-              target="_blank"
-              rel="noopener noreferrer"
-              className="vads-u-margin-left--0p5"
-              href="https://www.va.gov/privacy-policy/"
-            >
-              privacy policy
-            </a>
-            .
-          </p>
-        </>
+        <p>
+          I have read and accept the
+          <a
+            target="_blank"
+            rel="noopener noreferrer"
+            className="vads-u-margin-left--0p5"
+            href="https://www.va.gov/privacy-policy/"
+          >
+            privacy policy
+          </a>
```

Comment:
Maybe make this privacy policy a component? It looks like it's repeated a bunch in here.